custom_reports/object_management_api/definitions/annotation.yaml

```diff
@@ -0,0 +1,10 @@
+type: object
```

Comment:
As suggested @vanzi we should consider change response from blank object to object called "data" including actual response. Src: https://piwikpro.atlassian.net/wiki/spaces/PPMS/pages/7556169719/Public+API+Design+response+format+naming+urls